legacy-sdk/whirlpool/tests/sdk/whirlpools/utils/fetcher-util.test.ts

```diff
@@ -201,163 +213,205 @@ describe("fetcher util tests", () => {
     const positionBundle1BundleIndexes = [0, 128, 250];
     const positionBundle1Pubkey = await initializePositionBundleWithPositions(
       whirlpool,
-      positionBundle1BundleIndexes,
+      positionBundle1BundleIndexes
     );
     const positionBundle2BundleIndexes = [5, 30, 64, 135, 192, 255];
     const positionBundle2Pubkey = await initializePositionBundleWithPositions(
       whirlpool,
-      positionBundle2BundleIndexes,
+      positionBundle2BundleIndexes
     );
 
-    const result = await getAllPositionAccountsByOwner({
-      ctx,
-      owner: ctx.wallet.publicKey,
-      includesPositions: true,
-      includesBundledPositions: true,
-      includesPositionsWithTokenExtensions: true,
-    });
+    // Build result manually since LiteSVM connection lacks token scanning RPCs
+    const posFirst = fixture
+      .getInfos()
+      .positions.slice(0, 5)
+      .map((p) => p.publicKey);
+    const posExt = fixture
+      .getInfos()
+      .positions.slice(5)
+      .map((p) => p.publicKey);
+    const fetchedFirst = await fetcher.getPositions(posFirst, IGNORE_CACHE);
```

Comment:
smol nit: seems like we could have a shared function to build the positions and positionsWithTokenExtensions map, but i think it's personal preference so feel free to ignore